components/internal/ApplicantStatusBadge.tsx

```diff
@@ -0,0 +1,133 @@
+    | 'pending'
+    | 'rejected'
+    | 'expiring'
+    | 'expired'
+    | 'active'
+    | 'inactive';
+};
+
+export function ApplicantStatusBadge({ variant }: Props) {
+  /**
+   * Renders the applicant status badge content (icon + text)
+   * @param variant for specifying applicant status badge type
+   * @returns {JSX.Element} contents of the badge
+   */
+  const _renderBadgeContent = (variant: Props['variant']): JSX.Element => {
+    switch (variant) {
+      case 'completed':
+        return (
+          <>
+            <Image
+              src="/assets/completed-icon.svg"
+              alt="Completed Application Icon"
+              height={12}
+              width={12}
+            />
+            <Box as="span" ml="2">
+              Completed
+            </Box>
+          </>
+        );
+      case 'expiring':
+        return (
+          <>
+            <Image
+              src="/assets/expiring-icon.svg"
+              alt="Expiring Application Icon"
+              height={14}
+              width={14}
+            />
+            <Box as="span" ml="2">
+              Expiring &lt; 30 days
+            </Box>
+          </>
+        );
+      case 'expired':
+        return (
+          <>
+            <Image
+              src="/assets/expired-icon.svg"
+              alt="Expired Application Icon"
+              height={15}
+              width={16}
+            />
+            <Box as="span" ml="2">
+              Expired
+            </Box>
+          </>
+        );
+      case 'pending':
+        return (
+          <>
+            <Image
+              src="/assets/pending-icon.svg"
+              alt="Pending Application Icon"
+              height={12}
+              width={12}
+            />
+            <Box as="span" ml="2">
+              Pending
+            </Box>
+          </>
+        );
+      case 'rejected':
+        return (
+          <>
+            <Image
+              src="/assets/rejected-icon.svg"
+              alt="Rejected Application Icon"
+              height={12}
+              width={12}
+            />
+            <Box as="span" ml="2">
+              Rejected
+            </Box>
+          </>
+        );
+      case 'inProgress':
+        return (
+          <>
+            <Image
+              src="/assets/in-progress-icon.svg"
+              alt="In Progress Application Icon"
+              height={12}
+              width={12}
+            />
+            <Box as="span" ml="2">
+              In progress
+            </Box>
+          </>
+        );
+      case 'active':
+      default:
+        return (
+          <>
+            <Image
+              src="/assets/active-icon.svg"
+              alt="Active Application Icon"
+              height={12}
+              width={12}
+            />
+            <Box as="span" ml="2">
+              Active
+            </Box>
+          </>
+        );
+    }
+  };
+
+  return (
+    <Badge variant={variant}>
+      <Box ml="-1" mr="2">
```

Comment:
Are these pixels? If yes, could you please include the "px"? Just for consistency

Reply:
I believe this follows the spacing in `tools/theme/spacing.ts` which makes it equivalent to 
```tsx
      <Box ml="-4px" mr="8px">
```
Specifying the pixels seems more readable so I'll it to that instead